selfdrive/ui/qt/onroad.cc

```diff
@@ -402,6 +402,21 @@ void AnnotatedCameraWidget::drawHud(QPainter &p) {
 // Window that shows camera view and variety of
 // info drawn on top
 
+void AnnotatedCameraWidget::mousePressEvent(QMouseEvent* e) {
+  const int exp_btn_w = radius + bdr_s * 4;
+  const int exp_btn_h = radius + bdr_s * 3;
+
+  if (engageable) {
+    auto params = Params();
+    if (params.getBool("ExperimentalModeConfirmed") && e->x() > rect().right() - exp_btn_w && e->y() < exp_btn_h) {
```

Comment:
not sure the rect is going to be intuitive when you mis-tap, think a circle would be better

Reply:
A circular tap area? I don't think you'd notice that much in the car.